components/molecule/dropdownOption/src/index.scss

```diff
@@ -8,14 +8,20 @@ $bg-molecule-dropdown-option-selected: none !default;
 $fw-molecule-dropdown-option-selected-text: $fw-bold !default;
 $fz-molecule-dropdown-option-text: inherit !default;
 $fw-molecule-dropdown-option--highlight-mark: normal !default;
-$ln-molecule-dropdown-option-two-lines-text: $lh-m !default;
+$lh-molecule-dropdown-option-line-wrap-text: $lh-m !default;
```

Comment:
it might be retrocompatible, soo it should still be named
`$ln-molecule-dropdown-option-two-lines-text` but including this is deprecated and actual value is
`$lh-molecule-dropdown-option-line-wrap-text`and link the actual value